conjur-oss/README.md

```diff
@@ -120,6 +123,12 @@ $  DATA_KEY="$(docker run --rm cyberark/conjur data-key generate)"
 $  HELM_RELEASE=<helm-release>
```

Comment:
We should add `VERSION=<some default version>` here, and use it on line 135. Otherwise this example requires manual tweaking in combination with copying and pasting.

Reply:
otherwise @BradleyBoutcher does this look good? can you please help fix up the PR to unblock a helm tag?

Reply:
Sure thing, I'm just testing with it now, but I'll rebase the project and push that up